iaas/20220831.md

```diff
@@ -0,0 +1,40 @@
+# 2022-08-31
+
+## Participants - use of github handle is preferred
+* @fkr
+* @matfechner
+* @o-otte
+* @lossack
+* @ra-beer
+* @garloff
+* @master-caster (bzapiec)
+
+## Recurring items
+
+* Who is presenting in weekly community call and creates minutes from etherpad?
+  * @fkr
+  
+* Anything else to add to the agenda?
+  * Release 3 Notes!
+  * Ubuntu 22.04 & Yoga
+  
+
+## Agenda
+
+### Sprint Planning & Review
+
+* discussion about "not supported notice about dvr"
+  * we should provide architectural blueprints for how to deploy our ref implementation
+    * in several t-shirt sizes (suggested by @kurt)
+    * should be part of work on R4
+    * if one does not follow blueprints, there might be dragons
+    * -> action item for @fkr: write user story 
+
+* https://github.com/SovereignCloudStack/issues/issues/146
+  * still problems about updating to yoga
+  * to be verified by @ra-beer by thursday, if need be a separate call will be scheduled
+
+### Ubuntu 22.04
+
+* not supported, should be noted in Release Notes that Ubuntu 22.04 is only for the adventourous
```

Comment:
Spelling: adventurous.
/lgtm for the rest

Reply:
thanks Spelling 🐝 !